syntax="proto2";
option cc_generic_services = true;

package peleus_demo.internal_service.syn;

message MainRequest {
    required string message = 1;
};

message MainResponse {
    required string message = 1;
};

service MainService {
    rpc query(MainRequest) returns (MainResponse);
};
